crates/cheatnet/tests/cheatcodes/prank.rs

```diff
@@ -0,0 +1,23 @@
+use crate::{
+    assert_success,
+    common::{deploy_contract, get_felt_selector_from_name, state::create_cheatnet_state},
+};
+use cairo_felt::Felt252;
+use cheatnet::rpc::call_contract;
+use starknet_api::core::ContractAddress;
+
+#[test]
+fn prank_simple() {
+    let mut state = create_cheatnet_state();
+
+    let contract_address = deploy_contract(&mut state, "PrankChecker", vec![].as_slice());
+
+    state.start_prank(contract_address, ContractAddress::from(123_u128));
+
+    let selector = get_felt_selector_from_name("get_caller_address");
```

Comment:
I would pass a "str" to "call_contract" for convenience

Reply:
That would require forge to do a reverse transformation from the felt selector they have back to the string.